core/src/components/input/input.ionic.scss

```diff
@@ -19,10 +19,46 @@
 // Ionic Input Sizes
 // --------------------------------------------------
 
+:host(.input-size-medium) .native-wrapper {
+  min-height: 40px;
+}
+
 :host(.input-size-large) .native-wrapper {
   min-height: 48px;
 }
 
+// Target area
+// --------------------------------------------------
+:host .native-wrapper::after {
+  @include position(50%, 0, null, 0);
+
+  position: absolute;
+
+  height: 100%;
+  min-height: 48px;
+
+  transform: translateY(-50%);
+
+  content: "";
+
+  // Cursor should match the native input when hovering over the target area.
+  cursor: text;
+
+  z-index: 1;
+}
+
+::slotted([slot="start"]),
+::slotted([slot="end"]),
+.input-clear-icon {
+  /**
+   * The target area has a z-index of 1, so the slotted elements 
+   * should be higher. Otherwise, the slotted elements will not 
+   * be interactable. This is especially important for the clear
+   * button, which should be clickable.
+   */
+  z-index: 10;
```

Comment:
What is the reasoning behind `10` vs. `2` here?

Reply:
No reasoning. I just found 10 to be visually nicer. It works with 2, updating it now.